src/psalter.php

```diff
@@ -6,12 +6,17 @@
 use Psalm\IssueBuffer;
 use Psalm\Progress\DebugProgress;
 use Psalm\Progress\DefaultProgress;
+use function getMemoryLimitInBytes;
 
 // show all errors
 error_reporting(-1);
 ini_set('display_errors', '1');
 ini_set('display_startup_errors', '1');
-ini_set('memory_limit', '4096M');
+$memLimit = getMemoryLimitInBytes();
+// Magic number is 4096M in bytes
+if ($memLimit > 0 && $memLimit < 4294967296) {
```

Comment:
Maybe just use constant MIN_MEM_LIMIT and MIN_MEM_LIMIT_BYTES

Reply:
Maybe, but where would I define those, since this is the only place I check them? 

Plus, `MIN_MEM_LIMIT_BYTES` would make sense but I would have `MAX_MEM_LIMIT_BYTES` for the upper boundary which seems kind of counter intuitive since it is not actually the maximum of anything but rather the memory psalter wants to make sure to have. 

Not sure this gets readable any better with this.